admin/Rakefile

```diff
@@ -15,3 +15,17 @@ DummyApp::RakeTasks.new(
 )
 
 task test_app: 'db:reset'
+
+namespace :solidus_admin do
+  namespace :tailwind do
+    task :watch do
+      require "tailwindcss-rails"
+      require "solidus_admin/engine"
+      system "#{Tailwindcss::Engine.root.join("exe/tailwindcss")} \
+         -i #{SolidusAdmin::Engine.root.join("app/assets/stylesheets/application.tailwind.css")} \
+         -o #{SolidusAdmin::Engine.root.join("app/assets/builds/solidus_admin.css")} \
+         -c #{SolidusAdmin::Engine.root.join("config/tailwind.config.js")} \
+         --minify -w"
```

Comment:
Could it be beneficial to not minify the output in order to debug CSS issues? At least during rebuilding the admin?

Reply:
Honestly, I think that most debugging will happen through the browser development utilities. If we find something, we'll need to touch the HTML, not the compiled file. At this point, I see no value in skipping the minification, but we can reconsider if we think otherwise. Happy also to hear what others think :pray:

Reply:
If we start to feel the need of having it not minified when developing, we can change this anytime. 👍